packages/mdc-tab-scroller/foundation.ts

```diff
@@ -30,6 +30,11 @@ import {MDCTabScrollerRTLReverse} from './rtl-reverse-scroller';
 import {MDCTabScrollerRTL} from './rtl-scroller';
 import {MDCTabScrollerAnimation, MDCTabScrollerHorizontalEdges} from './types';
 
+interface ScrollOperation {
```

Comment:
This can be removed. `MDCTabScrollerAnimation` above provides the same interface.